security/rbac/scripts/enable-rbac-ksql-server.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+
+################################################################################
+# Overview
+################################################################################
+#
+################################################################################
+
+# Source library
+. ../../../utils/helper.sh
+. ./rbac_lib.sh
+
+check_env || exit 1
+check_cli_v2 || exit 1
+check_jq || exit 1
+
+##################################################
+# Initialize
+##################################################
+
+. ../config/local-demo.cfg
+ORIGINAL_CONFIGS_DIR=/tmp/original_configs
+DELTA_CONFIGS_DIR=../delta_configs
+FILENAME=ksql-server.properties
+create_temp_configs $CONFLUENT_HOME/etc/ksql/$FILENAME $ORIGINAL_CONFIGS_DIR/$FILENAME $DELTA_CONFIGS_DIR/${FILENAME}.delta
+
+# Log in to Metadata Server (MDS)
+login_mds $MDS
+
+##################################################
+# Administrative Functions
+# - Grant principal User:$ADMIN_KSQL the ResourceOwner role to Topic:_confluent-ksql-${KSQL_SERVICE_ID}_command_topic
+# - Grant principal User:$ADMIN_KSQL the ResourceOwner role to Topic:${KSQL_SERVICE_ID}_ksql_processing_log
+# - Start KSQL
+# - Grant principal User:$ADMIN_KSQL the SecurityAdmin role to the KSQL Cluster
+# - Grant principal User:$ADMIN_KSQL the ResourceOwner role to KsqlCluster:ksql-cluster
+##################################################
+
+# Get the Kafka cluster id
+get_cluster_id_kafka
+
+# Use the default KSQL app identifier
+KSQL_SERVICE_ID=rbac-ksql
+
+echo -e "\n# Grant principal User:$ADMIN_KSQL the ResourceOwner role to Topic:_confluent-ksql-${KSQL_SERVICE_ID}_command_topic"
+echo "confluent iam rolebinding create --principal User:$ADMIN_KSQL --role ResourceOwner --resource Topic:_confluent-ksql-${KSQL_SERVICE_ID}_command_topic --kafka-cluster-id $KAFKA_CLUSTER_ID"
```

Comment:
Might be slightly easier to just use `(set -x; confluent iam rolebinding create ...)` - will save you some duplicate lines. Not critical, of course :)

Reply:
@stan-confluent Thank you for this great suggestion and 💯 agree, and it will help address your other comment about using linebreaks (https://github.com/confluentinc/examples/pull/204#discussion_r304152771) because I wasn't using linebreaks because of the need to print.

I will address this in a separate jira/PR to cover *all* the code.